src/footer/Footer.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash-es';
+import { intlShape, injectIntl, FormattedMessage } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow,
+  Button,
+  Hyperlink,
+  Image,
+  TransitionReplace,
+} from '@edx/paragon';
+import { ExpandLess, ExpandMore, Help } from '@edx/paragon/icons';
+import messages from './messages';
+
+const Footer = ({
+  marketingBaseUrl,
+  termsOfServiceUrl,
+  privacyPolicyUrl,
+  supportEmail,
+  platformName,
+  lmsBaseUrl,
+  studioBaseUrl,
+  showAccessibilityPage,
+  // injected
+  intl,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
```

Comment:
I'm assuming the giant pile of jsx is not a choice you would make, but this is copied from the header package?